ml3d/datasets/customdataset.py

```diff
@@ -49,22 +57,30 @@ def get_data(self, idx):
             feat = np.array(data[:, 3:],
                             dtype=np.float32) if data.shape[1] > 3 else None
             labels = np.zeros((points.shape[0],), dtype=np.int32)
-
-        data = {'point': points, 'feat': feat, 'label': labels}
+            data = {'point': points, 'feat': feat, 'label': labels}
```

Comment:
Can you decrease a level of indentation here?

Reply:
I have decreased the indent.

Reply:
This is still incorrect for line `60`

Reply:
@ClarytyLLC please, check the indentation of "data". It should be outside the else clause.